mautic/mautic
# Email activity timeline fix

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #2313 #2231
| BC breaks? | 
| Deprecations? | 

**Note that all new features should have a related user and/or developer documentation PR in their respective repositories.** 

### Required
#### Description:
This PR includes failed emails in contact's timeline and also it fixes the messages shown for emails read and emails sent.
#### Steps to reproduce the bug:
1. have some emails with failed status and some with read status and some with only sent status
2. open the contact's timeline related to the emails sent. Notice that failed emails are treated as sent emails, and the descriptions under all of them are the same.

#### Steps to test this PR:
1. Apply this PR 
2. Notice now if an email is failed it says it has failed, and also if its been read it tell you correct info of when it was read.
